elements/lisk-transactions/src/legacy_cast_votes.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright © 2019 Lisk Foundation
```

Comment:
`2020`

Reply:
This is 2019 and its correct, this file will be removed once everything is implemented